### PR TITLE
Storefront update and initial support for Oracle Digital Assistant

### LINKED ISSUE #304: Support to use Oracle Digital Assistant(ODA) on the storefront
Need support to enable and configure Oracle Digital Assistant on the storefront.

Should not affect the default behavior to not enable ODA.

When enabled, need to load the web-sdk and have the necessary configurations to run the chatbot.

### PR DESCRIPTION
**Pull request by junior**
- Storefront general update
- Version Bumps
- NATS.io update to support arm64
- Oracle Digital Assistant initial support on the storefront (Enabled via variable)

Resolves #304 

>Note: This PR does not include updates on the stacks and support for ODA on the terraform scripts. This will come after this gets merged.

### CODE REVIEW

File: deploy/complete/helm-chart/setup/values.yaml [hunk 1]
```diff
@@ -120,7 +121,6 @@ grafana:
     - grafana-worldmap-panel
     - grafana-piechart-panel
     - btplc-status-dot-panel
-    - grafana-github-datasource
```
Comment (junior) [inline deploy/complete/helm-chart/setup/values.yaml:123]: Removal necessary as this plugin does not support arm64 and is not used by developers use case with MuShop

File: deploy/complete/helm-chart/mushop/charts/nats/values.yaml [hunk 2]
```diff
@@ -2,12 +2,12 @@ replicaCount: 1
 
 image:
   repository: nats
-  tag: 2.1.2
+  tag: 2.7.1
```
Comment (jeevanjoseph) [inline deploy/complete/helm-chart/mushop/charts/nats/values.yaml:5]: 👍

File: deploy/complete/helm-chart/mushop/charts/storefront/values.yaml [hunk 3]
```diff
@@ -15,7 +22,7 @@ resources:
 
 securityContext:
   runAsNonRoot: true
-  runAsUser: 10001
+  runAsUser: 101
```
Comment (jeevanjoseph) [inline deploy/complete/helm-chart/mushop/charts/storefront/values.yaml:25]: Why 101 , and not 1000 ?

File: deploy/complete/helm-chart/setup/values.yaml (hunk 1 above)
Comment (jeevanjoseph) [reply deploy/complete/helm-chart/setup/values.yaml:123]: 👍 . I’ve never seen anyone request this either

**Review by jeevanjoseph — APPROVED** [review]

File: deploy/complete/helm-chart/mushop/charts/storefront/values.yaml (hunk 3 above)
Comment (junior) [reply deploy/complete/helm-chart/mushop/charts/storefront/values.yaml:25]: New standard for nginx user on unprivileged images.

File: deploy/complete/helm-chart/setup/values.yaml (hunk 1 above)
Comment (junior) [reply deploy/complete/helm-chart/setup/values.yaml:123]: I was using to get stats on clones/forks. There's a new, but is out of context for MuShop deployment.

**Review by mvandervliet — APPROVED** [review]

**Pull request merged**